webapp/channels/src/components/sidebar/sidebar_list/sidebar_list.tsx

```diff
@@ -108,8 +108,11 @@ type Props = {
 type State = {
     showTopUnread: boolean;
     showBottomUnread: boolean;
+    autoHide: boolean;
 };
 
+let mouseMoveTimeout: NodeJS.Timeout;
```

Comment:
Could you make this a field of the `SidebarList` class? Strictly speaking, there should only be one `SidebarList` ever mounted at a time, but it's just a good practice to keep them inside the class in case someone copies this elsewhere